packages/framework/src/DataFixtures/DemoMultidomain/PricingGroupDataFixture.php

```diff
@@ -37,7 +37,14 @@ public function load(ObjectManager $manager)
 
         $pricingGroupData->name = 'Obyčejný zákazník';
         $domainId = 2;
-        $this->createPricingGroup($pricingGroupData, $domainId, self::PRICING_GROUP_ORDINARY_DOMAIN_2);
```

Comment:
Good question, in the first iteration it was. But Petr Heinz explained to me that it should not be because if you will add new domain and you will follow the proper instructions, there will be created Default pricing group for second domain in create-domains-data so there is no need for this call of createPricingGroup.